test/Sauce/Runner.js

```diff
@@ -14,16 +14,16 @@ if (!username) {
     {
       browserName: "chrome",
       platform: "Windows 10",
-      version: "57.0",
+      version: "59.0",
```

Comment:
WCT set's it by not selecting the version at all https://github.com/Starcounter/starcounter-include/blob/master/wct.conf.json#L12 I would suspect it just propagates this config to sauce

https://github.com/saucelabs/ci-sauce/issues/3
> Though, we realized that current version is used when we use an empty string for "version" in Selenium's DesiredCapabilities.

https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-BrowserVersion
> If you want to use the latest stable version of Google Chrome or Firefox that Sauce supports, you can use "version": "latest". You can also use "version": "latest-1" or "version": "latest-2", etc. to request the next most recent versions of a browser. For example, if the latest stable version of Chrome is 48, you can request "latest-2" to use Chrome 46.